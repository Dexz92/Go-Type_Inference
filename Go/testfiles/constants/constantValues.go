func main() {
  const a int = 2;
  a = 1;
  const b float64 = 3.14;
  b = 5;
  const c bool = false;
  c = true;

  const d = 5;
  d = 2;

  const e int = b + c;

}
