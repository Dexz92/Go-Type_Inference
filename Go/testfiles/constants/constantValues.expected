Error at line 3: Trying to assign a new value to a constant
Error at line 5: Trying to assign a new value to a constant
Error at line 7: Trying to assign a new value to a constant
Error at line 10: Trying to assign a new value to a constant
Error at line 12: Invalid operation: b + c (mismatched types: float and bool)
